p2p/src/channels/signaling/discovery/p2p_channels_signaling_discovery_reducer.rs

```diff
@@ -112,11 +121,11 @@ impl P2pChannelsSignalingDiscoveryState {
                 };
 
                 let dispatcher = state_context.into_dispatcher();
-                dispatcher.push(P2pChannelsSignalingDiscoveryEffectfulAction::MessageSend {
+                let message = SignalingDiscoveryChannelMsg::Discovered { target_public_key };
+                dispatcher.push(P2pChannelsEffectfulAction::SendMessage {
                     peer_id,
-                    message: SignalingDiscoveryChannelMsg::Discovered {
-                        target_public_key: target_public_key.clone(),
-                    },
+                    msg_id: MsgId::first(),
+                    msg: ChannelMsg::SignalingDiscovery(message),
```

Comment:
`ChannelMsg` derives `From` so you could just use `message.into()` here.

Reply:
I know, but it think it is easier to read.

Reply:
@0xMimir in cases where the message is not obvious from the context I agree with you.

But in this case in particular (and others like it), since the contents of the message are immediately visible in the context, does it really aid with readability?